.github/workflows/release.yml

```
@@ -106,6 +106,8 @@ jobs:
       artifact-suffix: ${{ matrix.artifact_suffix }}
 
   build-freebsd:
+    # Temporarily disabled until FreeBSD release builds can be validated.
+    if: ${{ false }}
```

actionlint (1.7.11)

[error] constant expression "false" in condition. remove the if: section

(if-cond)